src/components/AppSessionStateContext.tsx

```diff
@@ -42,11 +43,13 @@ const mapAppSessionStateToSimplifiedAppSessionState: (
     status,
     dopplerAccountName,
     unlayerUser: { id, signature },
+    lang,
   } = appSessionState;
   return {
     status,
     dopplerAccountName,
     unlayerUser: { id, signature },
+    lang,
```

Comment:
What do you think about comparing the value in `isTheSameSession`?

Here you have an example of the possible effect on user experience:

1. You open Doppler editors-webapp in a tab
2. You open the Doppler user's settings page in another tab
3. You change the language in Doppler user's settings page and save
4. When the polling interval force a session to refresh, depending on `isTheSameSession`, you will see the translation applied or not.

Reply:
The unit test it´s testing this.